sources/packages/api/src-sql/EducationPrograms/Add-field-of-study-code-column.sql

```diff
@@ -0,0 +1,7 @@
+-- Add field_of_study_code to education_programs table.
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN IF NOT EXISTS field_of_study_code INT NULL;
+
+COMMENT ON COLUMN sims.education_programs.field_of_study_code IS 'Education program field of study code.';
```

Comment:
Please add some business context to the comment, for instance, the below.
```
Federal field of study code calculated based on the CIP code and the credential type.
```
Please add the same to the entity model.

Reply:
I will update.

Reply:
Updated.